contrib/build-dokku.Dockerfile

```diff
@@ -41,7 +41,7 @@ RUN PLUGIN_MAKE_TARGET=${PLUGIN_MAKE_TARGET} \
 
 FROM builder as armhf
 
-COPY --from=amd64 /tmp /tmp
+COPY --from=armhf /tmp /tmp
```

Comment:
Please revert this. You're _in_ the `armhf` build stage. We're trying to grab the `/tmp` folder _from_ the `amd64` stage.

Maybe useful reading:

- https://www.docker.com/blog/advanced-dockerfiles-faster-builds-and-smaller-images-using-buildkit-and-multistage-builds/
- https://levelup.gitconnected.com/docker-multi-stage-builds-and-copy-from-other-images-3bf1a2b095e0

Reply:
Already did. Thanks.

Reply:
It wasn't clear why I would want to grab a `amd64` folder in an `armhf` stage.
That would make a useful comment in the dockerfile ;)